source/server/lsp-servers.ads

```diff
@@ -235,14 +281,15 @@ private
       Processing_Task : Processing_Task_Type (Server'Unchecked_Access);
       Output_Task     : Output_Task_Type (Server'Unchecked_Access);
       Input_Task      : Input_Task_Type (Server'Unchecked_Access);
-      Request_Map     : Request_Maps.Map;
```

Comment:
Request_Map is used only in Input_Task. There is no concurrency conflict there

Reply:
Indeed.

Reply:
To be clear: I still want to remove the pointer in Request_Maps, since I want to remove the type Message_Access - I'm doing this step by step.